source/src/cip/ciptypes.h

```diff
@@ -203,9 +201,7 @@ typedef struct {
  *
  */
 typedef struct {
-  EipUint8 path_size;
```

Comment:
The comment that the path_size should be 16 bit is wrong (sorry, my fault). It gives the length in 16 bit words and in all messages where it is used it is of size USINT (8 bit). Sorry again for the confusion